components/LegacyAnimation/LegacyAnimation.js

```diff
@@ -0,0 +1,13 @@
+import * as Styled from "./LegacyAnimation.styled";
+
+export default function LegacyAnimation({ color, opacity }) {
+  return (
+    <Styled.Container color={color} opacity={opacity}>
+      <Styled.BlobsContainer>
+        <Styled.Blob duration={"20s"} color={color} />
+        {/* {/* <Styled.Blob duration={"40s"} color={color} /> */}
```

Comment:
```suggestion
```

Reply:
should stay in for now, as we might use it later (work in progress :P)